lib/galaxy/tools/parameters/basic.py

```diff
@@ -163,9 +163,6 @@ def value_to_basic( self, value, app ):
         return self.to_string( value, app )
 
     def value_from_basic( self, value, app, ignore_errors=False ):
-        # HACK: Some things don't deal with unicode well, psycopg problem?
```

Comment:
> @mvdbeek Can you comment on this? Can you be confident this hack is no longer needed?

Doing a few standard things (running workflows, deleting datasets) it seems to work fine with postgresql-9.3 and postgresql-9.4.
pscycopg [explicitly supports unicode as well](http://initd.org/psycopg/docs/usage.html#unicode-handling), so ... I'm not sure what to do. I guess there is no chance to see what exactly was the reason for the hack :/